### DISCUSSION TITLE
Fighting offensive content by leveraging community feedback

### DISCUSSION THREAD

**Discussion by tk3369** [Humans-of-Julia/HoJBot.jl#2] [Ideas]
# Background 
A primary job of moderators is to ensure a comfortable environment for community members to interact with each other. For a diverse community with people coming from different culture, background, age group, etc., it is even more important to take this seriously.

The primary problems for manual moderation are:
1. It is sometimes difficult to decide whether something is offensive
2. The mods takes a lot of effort to discuss whether the content is appropriate or should be removed
3. It takes time to reach a consensus/decision before action can be taken

# Proposed approach
I propose that we automate this moderation work with the HoJBot. It could work as follows:

1. Member A posts something inappropriate
2. Member B felt uncomfortable and reacted with an `X` emoji.
3. Member C also felt uncomfortable and reacted with an `X` emoji.
4. etc.
5. When it reaches a threshold (e.g. 3 `X`'s) then the bot removes the post and notifies the mods & member A about the removal.

So basically, we let the community decides whether something is bad.

**Comment by mkschulze** [comment]
I like the idea and for sure it would be a cool feature. It could work with a proper introduction in the announcement channel. In general it would be nice to have a cheatsheet or quick reference about what we can do with all the bots. (another topic perhaps)

**Reply by rmsrosa** [reply]
I like it as well, but it would be better if it was anonymous, so people wouldn't be shy to complain.

**Reply by mkschulze** [reply]
agree!
